readme.md

```diff
@@ -18,19 +27,21 @@ Array of unprefixed property names (strings).
 module.exports.plugins = ['stylelint-value-no-numeric-constants'];
 
 module.exports.rules = {
-	'tradingview/value-no-numeric-constants': [['order', 'z-index'], {
+	'tradingview/value-no-numeric-constants': [{ properties: ['order', 'z-index']}, {
 		severity: 'warning'
 	}],
 }
 ```
 
 ## Values that triggers errors:
 
-Given `z-index` and `width` are specified in config:
+Config: `{ properties: ['width', 'z-index'], allowGt: 50, allowLt: 100}`
```

Comment:
So the lt and gt are applied to all properties at once.

What if I want to disable `z-index` entirely and restrict `width`? Or I want my `z-index` in range (-10 ... 10) and `order` to be (-2 ... 2)?